src/index.tsx

```diff
@@ -228,6 +229,31 @@ class Cropper extends React.Component<Props, State> {
     }
   }
 
+  getEmitCropSize = (
```

Comment:
I'm not a big fan of this `getEmit` pattern. Indeed, when calling this method, we are not sure that the returned cropSize will be actually used in the component. In the future, we could add some logic before calling `setState()` on line 228 and we could forget about this.

I'd rather to the checks right before setting the state. Even, if the `cropSize` might come from the props, I don't think this is an issue as the `onCropSizeChange` callback is still doing what is is supposed to do 🙂

Reply:
Very true, great call... I've updated the PR, moving the checks for cropSize changes to right before setting the state of the calculated cropSize value. If there is a difference between width or height, the callback prop will be called.